documentation/modules/exploit/multi/http/sugarcrm_webshell_cve_2023_22952.md

```diff
@@ -40,32 +40,53 @@ You can use this option to set the filename and extension of the webshell.
 This is handy if you want to test the webshell upload and execution with different file extensions (.phtml, .php7, .inc)
 to bypass any security settings on the Web and PHP server.
 
-## Scenarios
+### COMMAND
+This option provides the user to choose the PHP underlying shell command function to be used for execution.
+The choices are system(), passthru(), shell_exec() and exec() and it defaults to passthru().
+This option is only avialable when the target selected is either the Unix Command or Linux Dropper.
+For the native PHP target, by default the eval() function will be used for native PHP code execution.
```

Comment:
What if `eval()` is blocked on the target? Granted less likely given this is a construct that is built into the language though.